include/kframework/foundry.md

```diff
@@ -194,7 +194,7 @@ The configuration of the Foundry Cheat Codes is defined as follwing:
     - `<storageSlotSet>` - stores the storage whitelist containing pairs of addresses and storage indexes.
 
 ```k
-module FOUNDRY-CHEAT-CODES
+module FOUNDRY-CHEAT-CODES [symbolic]
```

Comment:
Declaring the `FOUNDRY-CHEAT-CODES` module `symbolic` will block the booster compilation. Even if the `symbolic` tag is removed, the compilation is still blocked by existential vars.